### DISCUSSION TITLE
Disable a periodic task when the schedule is removed from the env BEAT_SCHEDULE

### DISCUSSION THREAD

**Discussion by carlosdorneles-mb** [celery/django-celery-beat#559] [Q&A]
Hello everyone, is there any way for the module to detect when a schedule has been removed from the [BEAT_SCHEDULE](https://docs.celeryq.dev/en/stable/userguide/configuration.html#std-setting-beat_schedule) environment variable and automatically disable it in the database?

I know that today the module detects when new schedules in this variable and it automatically registers in the database, but I need something that works the other way around.

**Comment by m-r-k-f** [comment]
Hi, long time ago but still a topic, 
check out my solution
* #768
